sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/v2/DataSourceV2Relation.scala

```diff
@@ -23,7 +23,7 @@ import org.apache.spark.sql.sources.v2.reader._
 
 case class DataSourceV2Relation(
```

Comment:
Consider remove V2 in `DataSourceV2Relation` and `StreamingDataSourceV2Relation` ?

Reply:
This is internal, we can clean it up at any time. I wanna focus on public APIs in this PR.